test/integration/relay/connection.test.js

```diff
@@ -126,9 +127,19 @@ if (helper.sequelize.dialect.name === 'postgres') {
             values: {
               ID: {value: [this.Task.primaryKeyAttribute, 'ASC']},
               LATEST: {value: ['createdAt', 'DESC']},
+              CUSTOM: {value: ['updatedAt', 'DESC']},
               NAME: {value: ['name', 'ASC']}
             }
           }),
+          before: (options) => {
+            if (options.order[0][0] === 'updatedAt') {
+              options.order = Sequelize.literal(`
+                CASE
+                  WHEN completed = true THEN "createdAt"
+                  ELSE "updatedAt" End ASC`);
+            }
+            return options;
+          },
```

Comment:
Hmm, i believe i have that in one of my projects. Having an object shouldn't matter, the `value` in ENUM can be anything AFAIR.